api/features/views.py

```diff
@@ -290,12 +291,15 @@ def _filter_queryset(self, queryset: QuerySet) -> QuerySet:
         if "tags" in query_serializer.initial_data:
             if query_data.get("tags", "") == "":
                 queryset = queryset.filter(tags__isnull=True)
-            else:
+            elif query_data["tag_strategy"] == INTERSECTION:
```

Comment:
Feels slightly odd that we're checking if it's the default (as per the serializer) first, shouldn't we check for `UNION` first?

Reply:
I thought about it the other way around (i.e., check the default first since it's more likely), but if you're the reader and that's your conclusion then I'll take your word for it and switch them around.

Reply:
Ok I switched it around for you!

Reply:
Ha, yeah, your logic is also sound here but let's leave it in the switched order for now.